src/app/shared/post-capture-card/post-capture-card.component.ts

```diff
@@ -1,11 +1,16 @@
 import { HttpClient } from '@angular/common/http';
 import { Component, ElementRef, Input, OnInit, ViewChild } from '@angular/core';
+import { MatBottomSheet } from '@angular/material/bottom-sheet';
 import { Plugins } from '@capacitor/core';
 import { TranslocoService } from '@ngneat/transloco';
 import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
 import mergeImages from 'merge-images';
 import { BehaviorSubject } from 'rxjs';
-import { concatMap, first, map } from 'rxjs/operators';
+import { concatMap, first, map, tap } from 'rxjs/operators';
+import {
+  Option,
+  OptionsMenuComponent,
```

Comment:
This `OptionsMenuComponent` should __NOT__ be reused for other pages or components (e.g. `PostCaptureCardComponent`). You should create another component for `PostCaptureCardComponent`.